diff --git a/chrome/browser/chrome_browser_main.cc b/chrome/browser/chrome_browser_main.cc
index 64da513394753165966014e6cda6e20009b9b55f..d4e315aff74d17ad560b47f51eccb10864330243 100644
--- a/chrome/browser/chrome_browser_main.cc
+++ b/chrome/browser/chrome_browser_main.cc
@@ -1403,7 +1403,7 @@ int ChromeBrowserMainParts::PreMainMessageLoopRunImpl() {
 #endif  // BUILDFLAG(ENABLE_BACKGROUND_MODE)
   // Post-profile init ---------------------------------------------------------
 
-  TranslateService::Initialize();
+  TranslateService::Initialize(profile_);
   if (base::FeatureList::IsEnabled(features::kGeoLanguage) ||
       base::FeatureList::IsEnabled(language::kExplicitLanguageAsk) ||
       language::GetOverrideLanguageModel() ==
@@ -1591,10 +1591,12 @@ int ChromeBrowserMainParts::PreMainMessageLoopRunImpl() {
 
   PreBrowserStart();
 
+/*
   if (!parsed_command_line().HasSwitch(switches::kDisableComponentUpdate)) {
     component_updater::RegisterComponentsForUpdate(profile_->IsOffTheRecord(),
                                                    profile_->GetPrefs());
   }
+*/
 
   variations::VariationsService* variations_service =
       browser_process_->variations_service();
