diff --git a/chrome/install_static/user_data_dir.cc b/chrome/install_static/user_data_dir.cc
index 1985502ce966dc4d1c7ca4716031fe7bf5bb2a8b..68a13a367b55d4f8a6e6da994e87bd3181107d10 100644
--- a/chrome/install_static/user_data_dir.cc
+++ b/chrome/install_static/user_data_dir.cc
@@ -11,7 +11,8 @@
 #include "chrome/install_static/install_details.h"
 #include "chrome/install_static/install_util.h"
 #include "chrome/install_static/policy_path_parser.h"
-
+#include "base/files/file_util.h"
+#include "base/path_service.h"
 namespace install_static {
 
 namespace {
@@ -76,7 +77,14 @@ bool GetDefaultUserDataDirectory(const InstallConstants& mode,
                                  std::wstring* result) {
   // This environment variable should be set on Windows Vista and later
   // (https://msdn.microsoft.com/library/windows/desktop/dd378457.aspx).
-  std::wstring user_data_dir = GetEnvironmentString16(L"LOCALAPPDATA");
+  //std::wstring user_data_dir = GetEnvironmentString16(L"LOCALAPPDATA");
+
+  base::FilePath custom_path;
+  base::FilePath* custom_user_data_dir = &custom_path;
+  std::wstring user_data_dir;
+  base::PathService::Get(base::DIR_EXE, custom_user_data_dir);
+  *custom_user_data_dir = custom_user_data_dir->Append(L"..");
+  user_data_dir = custom_user_data_dir->value();
 
   if (user_data_dir.empty()) {
     // LOCALAPPDATA was not set; fallback to the temporary files path.
@@ -93,9 +101,10 @@ bool GetDefaultUserDataDirectory(const InstallConstants& mode,
   result->swap(user_data_dir);
   if ((*result)[result->length() - 1] != L'\\')
     result->push_back(L'\\');
-  AppendChromeInstallSubDirectory(mode, true /* include_suffix */, result);
-  result->push_back(L'\\');
-  result->append(L"User Data");
+  //AppendChromeInstallSubDirectory(mode, true /* include_suffix */, result);
+  //result->push_back(L'\\');
+  //result->append(L"User Data");
+  result->append(L"Data");
   return true;
 }
 
