diff --git a/chrome/browser/ui/webui/help/version_updater_win.cc b/chrome/browser/ui/webui/help/version_updater_win.cc
index be910614a2943a817d05f7b6ebfa1cf12d83ca79..1d1a615f42c702ce2965b1a8fbeb8cf7a5cd6318 100644
--- a/chrome/browser/ui/webui/help/version_updater_win.cc
+++ b/chrome/browser/ui/webui/help/version_updater_win.cc
@@ -31,8 +31,8 @@ void VersionUpdaterWin::CheckForUpdate(const StatusCallback& callback,
   // There is no supported integration with Google Update for Chromium.
   callback_ = callback;
 
-  callback_.Run(CHECKING, 0, false, false, std::string(), 0, base::string16());
-  DoBeginUpdateCheck(false /* !install_update_if_possible */);
+  callback_.Run(UPDATED, 0, false, false, std::string(), 0, base::string16());
+//  DoBeginUpdateCheck(false /* !install_update_if_possible */);
 }
 
 void VersionUpdaterWin::OnUpdateCheckComplete(
