diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 0e3a27eb469825c1bacf799b6e51dbe17d3667ce..5a695df0f0d95545d4b550c9e3ba0178d9b9de50 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -34,7 +34,9 @@ function addPrivacyChildRoutes(r) {
 
   // TODO(tommycli): Find a way to refactor these repetitive category
   // routes.
+      // <if expr="_google_chrome">
   r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
   r.SITE_SETTINGS_AR = r.SITE_SETTINGS.createChild('ar');
   r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
       r.SITE_SETTINGS.createChild('automaticDownloads');
@@ -101,7 +103,7 @@ function createBrowserSettingsRoutes() {
   r.SEARCH = r.BASIC.createSection('/search', 'search');
   if (!loadTimeData.getBoolean('isGuest') ||
       loadTimeData.getBoolean('isEphemeralGuestProfile')) {
-    r.PEOPLE = r.BASIC.createSection('/people', 'people');
+    r.PEOPLE = r.BASIC.createSection('/getStarted', 'getStarted');
     r.SYNC = r.PEOPLE.createChild('/syncSetup');
     r.SYNC_ADVANCED = r.SYNC.createChild('/syncSetup/advanced');
   }
