diff --git a/chrome/browser/resources/bookmarks/item.js b/chrome/browser/resources/bookmarks/item.js
index ff508f77c90a64a2cb4e2df4c0e2f7918d3b16ec..982c3cf94e4e4228b50c32ff43c573144bf68b47 100644
--- a/chrome/browser/resources/bookmarks/item.js
+++ b/chrome/browser/resources/bookmarks/item.js
@@ -196,7 +196,20 @@ Polymer({
       this.$.menuButton.focus();
     }
   },
-
+  /**
+   * @param {MouseEvent} e
+   * @private
+   */
+  onLinkClick_(e) {
+    if (!this.isSelectedItem_) {
+      this.selectThisItem_();
+    }
+    const commandManager = CommandManager.getInstance();
+    const itemSet = this.getState().selection.items;
+    if (commandManager.canExecute(Command.OPEN, itemSet)) {
+      commandManager.handle(Command.OPEN, itemSet);
+    }
+  },
   /**
    * @param {MouseEvent} e
    * @private
