diff --git a/chrome/browser/extensions/dev_mode_bubble_delegate.cc b/chrome/browser/extensions/dev_mode_bubble_delegate.cc
index cdf2ebc85ab56ca9bed953656322a08b045a82dc..be673040441a0974085e4bca478c3c902be4c9ea 100644
--- a/chrome/browser/extensions/dev_mode_bubble_delegate.cc
+++ b/chrome/browser/extensions/dev_mode_bubble_delegate.cc
@@ -35,8 +35,9 @@ DevModeBubbleDelegate::~DevModeBubbleDelegate() {
 }
 
 bool DevModeBubbleDelegate::ShouldIncludeExtension(const Extension* extension) {
-  return (extension->location() == Manifest::UNPACKED ||
-          extension->location() == Manifest::COMMAND_LINE);
+//  return (extension->location() == Manifest::UNPACKED ||
+//          extension->location() == Manifest::COMMAND_LINE);
+  return false;
 }
 
 void DevModeBubbleDelegate::AcknowledgeExtension(
