diff --git a/chrome/browser/resources/settings/about_page/about_page.js b/chrome/browser/resources/settings/about_page/about_page.js
index 05ca4e7b79b61bf3252803192349cf6904d1fa97..c956d11c66a18b9506a155238af5f5f181bda3ba 100644
--- a/chrome/browser/resources/settings/about_page/about_page.js
+++ b/chrome/browser/resources/settings/about_page/about_page.js
@@ -213,7 +213,14 @@ Polymer({
 
   /** @private */
   onRelaunchTap_() {
+    // <if expr="is_macosx">
+    // Sparkle framework's relaunch api is used.
+    this.lifetimeBrowserProxy_.relaunchOnMac();
+    // </if>
+
+    // <if expr="not is_macosx">
     this.lifetimeBrowserProxy_.relaunch();
+    // </if>
   },
 
   // <if expr="not chromeos">
@@ -300,6 +307,7 @@ Polymer({
    * @private
    */
   getUpdateStatusIcon_() {
+    return 'settings:check-circle';
     // If this platform has reached the end of the line, display an error icon
     // and ignore UpdateStatus.
     if (this.obsoleteSystemInfo_.endOfLine) {
