diff --git a/chrome/browser/resources/settings/about_page/about_page.js b/chrome/browser/resources/settings/about_page/about_page.js
index 05ca4e7b79b61bf3252803192349cf6904d1fa97..0e86cb24241324bba7b504972c900bcbfef4739a 100644
--- a/chrome/browser/resources/settings/about_page/about_page.js
+++ b/chrome/browser/resources/settings/about_page/about_page.js
@@ -124,11 +124,13 @@ Polymer({
 
     // <if expr="not chromeos">
     this.startListening_();
+/**
     if (Router.getInstance().getQueryParameters().get('checkForUpdate') ===
         'true') {
       this.onUpdateStatusChanged_({status: UpdateStatus.CHECKING});
       this.aboutBrowserProxy_.requestUpdate();
     }
+*/
     // </if>
   },
 
@@ -156,7 +158,7 @@ Polymer({
         'promotion-state-changed',
         this.onPromoteUpdaterStatusChanged_.bind(this));
     // </if>
-    this.aboutBrowserProxy_.refreshUpdateStatus();
+//    this.aboutBrowserProxy_.refreshUpdateStatus();
   },
 
   /**
@@ -213,7 +215,14 @@ Polymer({
 
   /** @private */
   onRelaunchTap_() {
+    // <if expr="is_macosx">
+    // Sparkle framework's relaunch api is used.
+    this.lifetimeBrowserProxy_.relaunchOnMac();
+    // </if>
+
+    // <if expr="not is_macosx">
     this.lifetimeBrowserProxy_.relaunch();
+    // </if>
   },
 
   // <if expr="not chromeos">
