diff --git a/components/sync/base/sync_prefs.cc b/components/sync/base/sync_prefs.cc
index 1e8d1e66b85b513b99fd30327cc3e5dcc4950eb8..505c0d3f308415adcd2b616ba425d1f0d7254554 100644
--- a/components/sync/base/sync_prefs.cc
+++ b/components/sync/base/sync_prefs.cc
@@ -255,7 +255,7 @@ void SyncPrefs::RegisterProfilePrefs(
   // Actual user-controlled preferences.
   registry->RegisterBooleanPref(prefs::kSyncFirstSetupComplete, false);
   registry->RegisterBooleanPref(prefs::kSyncRequested, false);
-  registry->RegisterBooleanPref(prefs::kSyncKeepEverythingSynced, true);
+  registry->RegisterBooleanPref(prefs::kSyncKeepEverythingSynced, false);
   for (UserSelectableType type : UserSelectableTypeSet::All()) {
     RegisterTypeSelectedPref(registry, type);
   }
