diff --git a/components/sync/base/sync_prefs.cc b/components/sync/base/sync_prefs.cc
index 735bc450f2c4633cfa7d607d1359c655d5a05e05..629cb6442cccd4bf2370b0824d1ad6a43b9cbaeb 100644
--- a/components/sync/base/sync_prefs.cc
+++ b/components/sync/base/sync_prefs.cc
@@ -152,7 +152,7 @@ void SyncPrefs::RegisterProfilePrefs(PrefRegistrySimple* registry) {
   // Actual user-controlled preferences.
   registry->RegisterBooleanPref(prefs::kSyncFirstSetupComplete, false);
   registry->RegisterBooleanPref(prefs::kSyncRequested, false);
-  registry->RegisterBooleanPref(prefs::kSyncKeepEverythingSynced, true);
+  registry->RegisterBooleanPref(prefs::kSyncKeepEverythingSynced, false);
   for (UserSelectableType type : UserSelectableTypeSet::All()) {
     RegisterTypeSelectedPref(registry, type);
   }
