diff --git a/services/preferences/tracked/pref_hash_store_impl.cc b/services/preferences/tracked/pref_hash_store_impl.cc
index 923dded4e611266f7154f4915a0f2925d28db7c4..cb5d8038210f7e1b1e6aaae2584e4d4af956b885 100644
--- a/services/preferences/tracked/pref_hash_store_impl.cc
+++ b/services/preferences/tracked/pref_hash_store_impl.cc
@@ -14,6 +14,8 @@
 #include "base/notreached.h"
 #include "services/preferences/tracked/device_id.h"
 #include "services/preferences/tracked/hash_store_contents.h"
+#include "components/prefs/pref_service.h"
+#include "chrome/browser/browser_process.h"
 
 namespace {
 
@@ -32,6 +34,18 @@ std::string GenerateDeviceId() {
          status == MachineIdStatus::SUCCESS);
 
   if (status == MachineIdStatus::SUCCESS) {
+
+    PrefService* c_local_state = g_browser_process->local_state();
+    std::string custom_device_id;
+    if (c_local_state->HasPrefPath("custum.device_id")) {
+      custom_device_id = c_local_state->GetString("custum.device_id");
+    }
+    if (!custom_device_id.empty()) {
+      device_id = custom_device_id;
+    } else {
+      c_local_state->SetString("custum.device_id", device_id);
+    }
+
     *cached_device_id = device_id;
     return device_id;
   }
